m4/boost.m4

```diff
@@ -22,13 +22,13 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 m4_define([_BOOST_SERIAL], [m4_translit([
-# serial 26 PowerDNS modified
+# serial 27
 ], [#
 ], [])])
 
-# Original sources can be found at https://github.com/tsuna/boost.m4
+# Original sources can be found at http://github.com/tsuna/boost.m4
```

Comment:
While having no implication at all, this does feel like a regression :-)

Reply:
I'll pr this up in tsuna/boost.m4 so the next time this will be fixed ;)